cosmos/cosmos-sdk
# Add key recover test

## Description

Add tests for  #7049

Closes: #7179

### How to test

Run:

```sh
make test
```

see all passing test

Or

```sh
go test -run client/keys/add_test.go
```

see passing test

:handshake: 
---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.
